tests/containers/containerd_rmt.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart in a k3s
+# - install k3s, kubectl and helm
+# - test helm repo add, update, search and show all
+# - add bitnami repo
+# - test helm install with apache helm chart
+# - test helm list
+# - check the correct deployment of the helm chart
+# - cleanup system (helm and k3s)
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils;
+use version_utils qw(is_sle);
+use serial_terminal 'select_serial_terminal';
+use registration qw(add_suseconnect_product get_addon_fullname);
+use containers::k8s;
+
+sub run {
+    my ($self) = @_;
+
+    select_serial_terminal;
+    systemctl 'stop firewalld';
+    install_k3s();
+    install_kubectl();
+    install_helm();
+
+    my $rmt_helm = get_var('RMTTEST_REPO', 'https://gitlab.suse.de/brunoleon/rmt-helm/-/archive/main/rmt-helm-main.zip');
+    zypper_call('in git-core', exitcode => [0, 65, 107]);
+    # pull in the testsuite
+    assert_script_run("wget --quiet --no-check-certificate " . $rmt_helm);
+    assert_script_run("unzip rmt-helm-main.zip");
+    my $myvalue = get_var('MY_VALUE', 'https://gitlab.suse.de/QA-APAC-I/testing/-/raw/master/data/rmtcontainer/myvalue.yaml');
+    assert_script_run("wget --no-check-certificate " . $myvalue);
+    assert_script_run('cat myvalue.yaml');
+    assert_script_run("helm install rmt ./rmt-helm-main -f myvalue.yaml");
+    assert_script_run("helm list");
+    my @out = split(' ', script_output("kubectl get pods | grep rmt-app"));
+    while (1) {
```

Comment:
I find this approach of infinite loop a bit tricky. The pod is being initialized at the time, the test is already trying to gather logs from the pod.
Maybe it would be better to be more granular and split these actions to:

1) wait while the pod successfully starts
2) query the logs whether the repos inside the container have been enabled.

Reply:
I want to wait time exactly. 
If only wait pod started successfully. then, we still need wait until rmt-app mirrored the expected repos.
We need get two wait times. 
Use this loop, we can get exact time.

Reply:
The wait time is not constant, if we need mirror more repos at 'myvalue.yaml', we need wait more time. So I use loop to wait the expected information in the log file.

Reply:
but this loop is dangerous, there is no timeout.

Reply:
Yes, I will set timeout .